test/common/json/json_fuzz_test.cc

```diff
@@ -23,6 +23,16 @@ DEFINE_FUZZER(const uint8_t* buf, size_t len) {
     ProtobufWkt::Struct message2;
     MessageUtil::loadFromJson(MessageUtil::getJsonStringFromMessage(message), message2);
     FUZZ_ASSERT(TestUtility::protoEqual(message, message2));
+
+    // MessageUtil::getYamlStringFromMessage automatically convert types, so we have to do another
+    // round-trip.
+    std::string yaml = MessageUtil::getYamlStringFromMessage(message);
+    ProtobufWkt::Struct yaml_message;
+    MessageUtil::loadFromYaml(yaml, yaml_message);
+
+    ProtobufWkt::Struct message3;
+    MessageUtil::loadFromYaml(MessageUtil::getYamlStringFromMessage(yaml_message), message3);
```

Comment:
Have you run this fuzzer locally manually to get some burn time?

Reply:
Yes, and ran fuzzit (also in CI) corpus too.